src/GenSVM.py

```diff
@@ -0,0 +1,73 @@
+import pandas as pd
+from sklearn.model_selection import GridSearchCV, train_test_split
+from sklearn.metrics import confusion_matrix, mean_absolute_error, mean_squared_error, cohen_kappa_score
+from sklearn.svm import SVC
+
+TEST_RATIO = 0.2
+
+
+class SVMModel:
+
+    def __init__(self):
+        self.data = pd.DataFrame()
+        self.target = pd.DataFrame()
+        self.classifier = SVC()
+        self.mae = 0
+        self.mse = 0
+        self.matrix = None
+
+    def load_data(self, filepath):
+        self.data = pd.read_csv(filepath)
+        self.data.head()
+        return self.data
+
+    def load_target(self, filepath):
+        self.target = pd.read_csv(filepath)
+        self.target.head()
+        return self.target
+
+    def get_sets(self):
+        # TODO
```

Comment:
Maybe throwing a `NotImplemented` exception would be better in case we forget to implement and being confused with later bugs

Reply:
Done,